contracts/script/examples/cross-token/.env.template

```diff
@@ -0,0 +1,15 @@
+export PRIVATE_KEY=
+export PUBLIC_ADDRESS=
```

Comment:
sorry, was debugging this so could not help but again, what's this public address? Is it derived from PRIVATE_KEY? If yes, then we dont need to config this?

Reply:
> sorry, was debugging this so could not help but again, what's this public address? Is it derived from PRIVATE_KEY? If yes, then we dont need to config this?

Yes that is correct. Can you suggest a way to get the public address from a private key? I don't see a straightforward way to do it using forge's tooling

Reply:
Check this out: https://book.getfoundry.sh/reference/cast/cast-wallet-address

Reply:
Or are you using this within a Forge script?

Reply:
Thank you!!!! This was irking me that I didn't see a way to do this in the documentation. Thanks for finding. I will clean up where we have PUBLIC_ADDRESS and use `cast wallet address --private-key $PRIVATE_KEY` instead 🎉

Reply:
coded here 81060d6b3eafdc5a38f9cccc2913957263cc09c1